nifi-nar-bundles/nifi-standard-services/nifi-oauth2-provider-api/src/main/java/org/apache/nifi/oauth2/AccessDetails.java

```diff
@@ -17,53 +17,64 @@
 
 package org.apache.nifi.oauth2;
 
-public class AccessToken {
+public class AccessDetails {
     private String accessToken;
     private String refreshToken;
     private String tokenType;
-    private Integer expires;
-    private String scope;
+    private Integer expiresIn;
+    private String scopes;
 
-    private Long fetchTime;
+    private final Long fetchTime;
```

Comment:
This should be changed to a `long` to avoid unexpected NullPointerExceptions in `isExpired()`. It would probably be better to represent that as a `java.time.Instant`.